src/behaviours/application-update/installing-update.test.ts

```diff
@@ -113,6 +113,12 @@ describe("installing update", () => {
         it("renders", () => {
           expect(rendered.baseElement).toMatchSnapshot();
         });
+
+        it.skip("when 5 seconds elapses, clears the notification to the user", () => {
```

Comment:
could not get this to work, maybe you can figure it out @jansav ?

Reply:
It is actually working in isolation. I've have same trouble in my branch and this just gave me idea what is causing the trouble. It seems that `now()` from `mobx-utils` is using shared global state. I'm gonna double check and figure it out.

Reply:
So I managed to create minimal unit test to prove the point:

```typescript
import { computed, observe } from "mobx";
import { now as reactiveNow } from "mobx-utils";

describe("reactiveNow", () => {
  let actual: boolean;

  beforeEach(() => {
    jest.useFakeTimers();

    jest.setSystemTime(new Date("2015-10-21T07:28:00Z"));

    const someComputed = computed(() => {
      const currentTimestamp = reactiveNow(1000);

      return currentTimestamp > new Date("2015-10-21T07:28:00Z").getTime();
    });

    observe(someComputed, (changed) => {
      actual = changed.newValue as boolean;
    }, true);
  });

  it("when time passes, returns true", () => {
    jest.advanceTimersByTime(1000);

    expect(actual).toBe(true);
  });

  it("does not share the state from previous test", () => {
    expect(actual).toBe(false);
  });
});
```

Key here is that if you focus on either of tests individually, they work. But since both are running, the latter fails. 

I'll figure out the solution next.

Reply:
Fixed the issue here: https://github.com/lensapp/lens/pull/5764

It was more than earlier described.